scripts/release_scripts/update_changelog_and_credits.py

```diff
@@ -435,7 +438,11 @@ def create_branch(repo_fork, target_branch, github_username):
         'expand=1' % (github_username, target_branch))
     python_utils.PRINT(
         'Pushed changes to Github. '
-        'Please create a pull request from the %s branch' % target_branch)
+        'Please create a pull request from the %s branch\n\n'
+        'Note: PR title should be exactly: '
```

Comment:
I think you might be able to just provide a URL with the title in it? Here's an example:

https://github.com/oppia/oppia/compare/develop...feature-branch?quick_pull=1&title=this-is-awesome

(change the title and the source/target branch names appropriately).

Reply:
Done.